google/cloud/storage/_helpers.py

```diff
@@ -134,6 +136,14 @@ def _query_params(self):
             params["userProject"] = self.user_project
         return params
 
+
+    def _call_api(self, client, retry, **kwargs):
```

Comment:
Should retry be a kwarg? Do callers explicitly have to pass in None if they don't want a retry? Would be good to clarify here.